Python/pywarpx/picmi.py

```diff
@@ -416,6 +416,10 @@ def initialize_inputs(self, species_number, layout, species, density_scale):
         species.flux_normal_axis = self.flux_normal_axis
         species.surface_flux_pos = self.surface_flux_position
         species.flux_direction = self.flux_direction
+        if self.flux_tmin is not None:
```

Comment:
The check `is not None` is not needed. If parameters are set to None, they are ignored.